pkg/model/estimator/local/ratio.go

```diff
@@ -104,7 +104,7 @@ func (r *RatioPowerModel) GetPlatformPower(isIdlePower bool) ([]float64, error)
 		} else {
 			processPower = r.getPowerByRatio(processIdx, int(PlatformUsageMetric), int(PlatformDynPower), numProcesses)
```

Comment:
It may make more sense here to fix the return value of `r.getPowerByRatio` given the float is immediately converted to uint64 here.